tutorials/time-series-visualization-with-altair/tutorial.ipynb

```diff
@@ -0,0 +1,2133 @@
+{
```

Comment:
<p>Should "Python" be changed to "pandas" or "Altair"? (Python is used to control Earth Engine in Steps 1-3.)</p><p> </p><p>Should "first example" be "first dataset"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Re: steps - I wanted to give folks a sense for what was happening in Earth Engine and Python. What I really meant was server-side and client-side. I've updated to use "server-side Earth Engine" and "client-side &lt;py library&gt;"</p><p> </p><p>Re: "first example" &gt; "first dataset" - done.</p><p> </p>

Reply:
<p>lgtm</p><p> </p>